monkey/monkey_island/cc/ui/src/components/pages/ReportPage.js

```diff
@@ -56,6 +58,19 @@ class ReportPageComponent extends AuthComponent {
       this.getZeroTrustReportFromServer().then((ztReport) => {
         this.setState({zeroTrustReport: ztReport})
       });
+      this.setState({
+        ransomwareReport: {'report': '',
+                           'show': true}})
+      // this.authFetch('/api/report/ransomware')
+      //   .then(res => res.json())
+      //   .then(res => {
+      //     this.setState({
+      //       ransomwareReport: res
+      //     });
+      //   });
```

Comment:
This assumes that Monkey Island will instruct the frontend whether or not it should show the report, which isn't the Island's responsibility. See https://github.com/guardicore/monkey/pull/1297#discussion_r663895620

This component will probably need to show the tab if there er any data in the report, or hide it otherwise.